server/sonar-db-dao/src/main/resources/org/sonar/db/organization/OrganizationMapper.xml

```diff
@@ -187,6 +187,13 @@
     org.uuid = #{uuid, jdbcType=VARCHAR}
   </select>
 
+  <select id="selectNewProjectPrivateByUuid" resultType="Boolean">
+    select org.new_project_private as "newProjectPrivate"
```

Comment:
`as "newProjectPrivate"` is useless when no mapping to an object field is required